webapp/src/client/httpClient.ts

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Client4} from 'mattermost-redux/client';
+import {ClientError} from 'mattermost-redux/client/client4';
+
+class HttpClient {
```

Comment:
0/5 but would it make sense for this to extend `Client4` so that you could use `doFetch` instead of reimplementing that? You wouldn't have the URL set automatically by the web app, but you'd get the rest of that as-is

Reply:
I don't think that class is exported, but yeah, will check it out later.

Reply:
It should be available from `@mattermost/client` instead of getting it from `mattermost-redux`

Reply:
Will do later if needed in interest of time.